src/domain/accounts/domain/src/entities/access_token.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+///////////////////////////////////////////////////////////////////////////////
+
+use chrono::{DateTime, Utc};
+use jsonwebtoken::TokenData;
+use opendatafabric::{AccountID, Multihash};
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct KamuAccessToken {
+    pub prefix: String,
+    pub id: Uuid,
+    pub random_bytes: [u8; 16],
+    pub random_bytes_hash: Multihash,
```

Comment:
what's the connection of `Multihash` and token hashing?

Reply:
I just reused it for hashing. But I agree it is overhead. Changed to bytes